src/run/run.c

```diff
@@ -2647,6 +2669,36 @@ static int start_transient_service(sd_bus *bus) {
                 if (r < 0)
                         return r;
 
+                sd_event_source **forward_signal_sources = NULL;
+                size_t n_forward_signal_sources = 0;
+                CLEANUP_ARRAY(forward_signal_sources, n_forward_signal_sources, event_source_unref_many);
+
+                /* There's no guarantee that we're permitted to forward signals to the service main process,
+                 * but there's no harm in trying. */
+                r = event_forward_signals(
+                        c.event,
+                        &c.pidref,
+                        pty_forward_signals, ELEMENTSOF(pty_forward_signals),
+                        &forward_signal_sources, &n_forward_signal_sources);
```

Comment:
we usually indent func params on line-broken func calls with double the usual indentation, i.e. 16 spaces